odoo/addons/base/models/res_users.py

```diff
@@ -611,6 +611,11 @@ def create(self, vals_list):
         return users
 
     def write(self, values):
+        if 'avatar_1920' in values and len(values) == 1:
```

Comment:
What's the purpose of all this ? 

If I write only on the avatar something that is different than the default avatar, then I write this value on another field instead ? 

Could you please comment + fix the tracebacks + write a meaningful commit message ? See: https://www.odoo.com/documentation/17.0/contributing/development/git_guidelines.html#commit-message-structure